css/style.css

```diff
@@ -17,3 +17,156 @@ body {
  */
 
 
+
+ /* MY COMMENTS:
+ Navigation Styles */
+
+header {
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 100px;
+    display: flex; 
+}
+
+.logo-img {
+    max-height: 32px;
+    max-width: 32px;
+    display: block;
+}
+
+ul {
+    list-style: none;
+    display: flex;
+    gap: 10px;  
+}
+
+ul li a {
+    color: rgb(158, 156, 156);
+    text-decoration: none;
+    padding: 15px;
+    
+}
+
+ul li a:hover {
+    color: #CE5C08;
+}
+
+.nav-head {
+    color: rgb(63, 57, 57);
+    font-weight: 500;
+    font-size: 14px;
+    
+}
+
+/* Hero Styles*/
+
+.section1 {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    color: rgb(238, 230, 230);
+    background-image: url("../img/first-background.jpg");
+    background-repeat:no-repeat;
+    background-size: cover;
+    height: 500px;
+    font-size: 22px;
+    font-weight: 12;
```

Comment:
font-weights are usually "lighter", "normal", "bold" or in 100s eg "100", "200" etc